examples/embedded/src/main/java/org/eclipse/jetty/embedded/ExampleServer.java

```diff
@@ -45,6 +45,12 @@ public static void main(String[] args) throws Exception
         handlers.setHandlers(new Handler[]{context, new DefaultHandler()});
         server.setHandler(handlers);
 
+        return server;
+    }
+
+    public static void main(String[] args) throws Exception
+    {
+        Server server = createServer(8080);
```

Comment:
Having made the port configurable, you may as well set it from a jetty.port system property that defaults to 8080

Reply:
If we did that we'd wind up with ...

``` java

int port = Integer.parseInt(System.getProperty("jetty.port", "8080"));
Server server = createServer(port);
```

Is that really what you want?

Reply:
yep - but I'm only +0 on it. if you don't want it don't do it.

Reply:
I've gone ahead and implemented the configurable port.